templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -341,58 +288,136 @@ contents:
         else
-          nmcli c add type ovs-interface slave-type ovs-port conn.interface br-ex master ovs-port-br-ex con-name \
-            ovs-if-br-ex 802-3-ethernet.mtu ${iface_mtu} 802-3-ethernet.cloned-mac-address ${iface_mac} \
+          nmcli c add type ovs-interface slave-type ovs-port conn.interface "$bridge_name" master "$ovs_port" con-name \
+            "$ovs_interface" 802-3-ethernet.mtu ${iface_mtu} 802-3-ethernet.cloned-mac-address ${iface_mac} \
             ipv4.route-metric 100 ipv6.route-metric 100
         fi
       fi
-
       # wait for DHCP to finish, verify connection is up
       counter=0
+      configured=false
+
       while [ $counter -lt 5 ]; do
         sleep 5
         # check if connection is active
-        if nmcli --fields GENERAL.STATE conn show ovs-if-br-ex | grep -i "activated"; then
+        if nmcli --fields GENERAL.STATE conn show "$ovs_interface" | grep -i "activated"; then
           echo "OVS successfully configured"
+          update_nm_conn_files "$bridge_name" "$port_name"
           copy_nm_conn_files
-          ip a show br-ex
+          ip a show "$bridge_name"
           ip route show
           nmcli c show
           configure_driver_options ${iface}
-          exit 0
+          configured=true
+          break
         fi
         counter=$((counter+1))
       done
-
       echo "WARN: OVS did not succesfully activate NM connection. Attempting to bring up connections"
       counter=0
       while [ $counter -lt 5 ]; do
-        if nmcli conn up ovs-if-br-ex; then
+        if nmcli conn up "$ovs_interface"; then
           echo "OVS successfully configured"
+          update_nm_conn_files "$bridge_name" "$port_name"
           copy_nm_conn_files
-          ip a show br-ex
+          ip a show "$bridge_name"
           ip route show
           nmcli c show
           configure_driver_options ${iface}
-          exit 0
+          configured=true
+          break
         fi
         sleep 5
         counter=$((counter+1))
       done
+      if [ $configured != true ]; then
+        echo "ERROR: Failed to activate $ovs_interface NM connection"
+        exit 1
+      fi
+    }
 
-      echo "ERROR: Failed to activate ovs-if-br-ex NM connection"
+    if ! rpm -qa | grep -q openvswitch; then
+      echo "Warning: Openvswitch package is not installed!"
       exit 1
+    fi
+    echo "Current routing and connection state:"
+    ip route show
+    nmcli c show
+    if [ "$1" == "OVNKubernetes" ]; then
+      # Configures NICs onto OVS bridge "br-ex"
+      # Configuration is either auto-detected or provided through a config file written already in Network Manager
+      # key files under /etc/NetworkManager/system-connections/
+      # Managing key files is outside of the scope of this script
+      # if the interface is of type vmxnet3 add multicast capability for that driver
+      # REMOVEME: Once BZ:1854355 is fixed, this needs to get removed.
+      extra_bridge_file='/etc/ovnk/extra_bridge'
```

Comment:
your commit says /var/run/extra_bridge, but this is different. Honestly I prefer it being under ovnk/

Reply:
Ah yes, it was originally under /var/run but then it seemed I can't deploy a file there as ignition, so I pivoted to /etc/ovnk/extra_bridge